test/session_expiration.spec.js

```diff
@@ -0,0 +1,32 @@
+const { expect } = require('chai');
+const supertest = require('supertest');
+const app = require('../app');
+const request = supertest(app);
+const { runFactory, authorizationFactory } = require('./helpers');
+const createUsers = runFactory('user');
+const { AUTHENTICATION_ERROR } = require('../app/errors');
+
+describe('POST /users/sessions', () => {
+  it('should respond with expiration time', () =>
+    createUsers(1)
+      .then(([{ email }]) => request.post('/users/sessions').send({ email, password: '1234567ocho' }))
+      .then(response => {
+        expect(response.statusCode).to.equal(200);
+        expect(response.body).to.have.property('tokenExpiresAt');
+      }));
+
+  it('should fail because token has expire', () =>
+    createUsers(1)
+      .then(([{ id }]) => authorizationFactory.regular(id))
```

Comment:
Several times i see this, create user and get token, maybe you can build a single function for this

Reply:
I actually did this in the previous PR, I just wanted to let you know in case it ended up being a little confusing.